modules/search/instant-search/components/gridicon/index.jsx

```diff
@@ -0,0 +1,380 @@
+/** @jsx h **/
+
+/* !!!
+This is a fork of the Jetpack Gridicon code:
+ https://github.com/Automattic/jetpack/blob/f8078c2cd12ac508334da2fb08e37a92cf283c14/_inc/client/components/gridicon/index.jsx
+
+It has been modified to work with Preact, and only includes the icons that we need.
```

Comment:
What are the icons we'll really need in the end? It looks like you've used tag, folder, and comment so far. Is that correct?

If we only need a handful of icons, could we just insert the SVG right away, and forget about Gridicons?

Reply:
I'd be tempted to keep things simple for now and only use a few SVGs when we need them. We can always start pulling Gridicons later if it seems like we'd need them. Right now we're not benefiting from the whole lib anyway since we're using a copy.

That said, I'll let you decide based on what we're going to need in the near future. You're in a better position than me to know that :)